clients/java/signalr/src/test/java/JsonHubProtocolTest.java

```diff
@@ -84,19 +84,6 @@ public void ParseSingleUnsupportedStreamInvocationMessage() {
         HubMessage[] messages = jsonHubProtocol.parseMessages(stringifiedMessage);
     }
 
-    @Test
-    public void ParseHandshakeResponsePlusMessage() {
```

Comment:
You should add a HubConnection test for receiving a HandshakeResponse + a message